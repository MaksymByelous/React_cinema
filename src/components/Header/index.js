import React from 'react';
import Styles from './styles.scss';
import AppStyles from '../../containers/App/styles.scss';
import moonIcon from '../../theme/assets/moon-icon.png';

const Header = () => (
    <header>
        <div className = { AppStyles.content }>
            <div>
                <img src = { moonIcon } />
                <div className = { Styles.movies }>
                    Movies
                </div>
            </div>
        </div>
    </header>
);

export default Header;
