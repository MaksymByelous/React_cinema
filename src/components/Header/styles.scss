header {
    background: #474e5a;
    padding: 5px 0;
    .content {
        display: flex;
        flex-flow: row wrap;
        justify-content: space-between;
        position: relative;
        max-width: 53.125rem;
        padding: 0.5rem;
        left: 50%;
        transform: translate(-50%, 0);
        overflow: hidden;
    }
    .movies {
      display: inline-block;
      padding: 5px 0 0 5px;
      color: white;
    }
    .login {
      // display: flex;
      border: 1px solid grey;
      border-radius: 5px;
      padding: 5px;
      color: lightgrey;
      .account {
        @media screen and (max-width: 750px) {
          display: none;
        }
      }
      i {
        color: white;
        margin-left: 5px;
        @media screen and (max-width: 750px) {
          margin-left: 0;
        }
      }
    }
}