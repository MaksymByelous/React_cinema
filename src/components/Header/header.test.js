import React from 'react';
import dom from 'react-test-renderer';
import Header from './';

const renderTree = dom.create(
    <Header />
).toJSON();

test(
    'header should correspond to snapshot', () => {
        expect(renderTree).toMatchSnapshot();
    }
);
