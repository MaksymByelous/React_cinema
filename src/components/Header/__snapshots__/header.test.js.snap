// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header should correspond to snapshot 1`] = `
<header>
  <div
    className={undefined}
  >
    <div>
      <img
        src="moon-icon.png"
      />
      <div
        className={undefined}
      >
        Movies
      </div>
    </div>
  </div>
</header>
`;
